src/elements/accordion/src/index.tsx

```diff
@@ -107,3 +112,20 @@ Accordion.Group = ({ children }) => {
     </AccordionContext.Provider>
   )
 }
+
+Accordion.Title = ({ children, as = 'h2' }) => {
+  const {
+    theme: { accordion: accordionTheme = {} }
+  }: any = useThemeUI()
+
+  return (
+    <Styled.h4
+      as={as}
+      sx={{
+        marginBottom: accordionTheme?.base?.title?.marginBottom
```

Comment:
Wouldn't this be better as `variant: 'theme.accordion.accordionTheme.base.title'`?

Reply:
It was already available as accordionTheme in this file

Reply:
We should remove that and use variants as Callum suggested IMO - that's the pattern we are using everywhere else